.github/workflows/CI.yml

```diff
@@ -30,3 +30,4 @@ jobs:
     - name: Build and Run Tests
       run: |
           ./gradlew build
+          ./gradlew help --task publishPluginZipPublicationToZipStagingRepository
```

Comment:
Looks like we just need `./gradlew publishPluginZipPublicationToZipStagingRepository`?

Reply:
@reta, I wanted to check opinions here, yes even this `./gradlew publishPluginZipPublicationToZipStagingRepository` can be added, but do we need to, I see there are multiple custom plugins already added to the template, but we dont run a plugin specific task, just `./gradlew build` exists, do we even need to add `publishPluginZipPublicationToZipStagingRepository` to test the CI?

Reply:
Ah I see, I think for the sake of checking that publishing works, we could publish to local repo: `publishPluginZipPublicationToZipLocalRepository`

Reply:
Yes we can add `./gradlew publishPluginZipPublicationToZipStagingRepository`, also moving forward its better we add some validation by running the custom plugin tasks, so that way we can some evidence that the custom plugins task actually work, I will add `./gradlew publishPluginZipPublicationToZipStagingRepository` to the CI.yaml.

Reply:
:+1: